src/extension.ts

```diff
@@ -62,7 +63,7 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
             item.text = localize("liberty.ls.thumbs.up");
             item.tooltip = localize("liberty.ls.started");
             toggleItem(window.activeTextEditor, item);
-    
+            handleWorkspaceSaveInProgress(getProjectProvider(context));
```

Comment:
@JoshwinThomasIBM Instead of passing 'getProjectProvider(context)'  as a parameter, can we call this method directly inside `handleWorkspaceSaveInProgress`, similar to how it is done in `registerCommands`?

Reply:
you mean to  call handleWorkspaceSaveInProgress by passing context as parameter and inside handleWorkspaceSaveInProgress get the project provider instance ?

Reply:
yes

Reply:
Changes are done as suggested